### ISSUE TITLE
Not Working in Python 3.11.2 & googletrans 3.0.0

### ISSUE THREAD

**Issue by Bhaverohit** [anshul7898/Gui-Tkinter-Based-Google-Translation#1]
I'm using Python==3.11.2, googletrans==3.0.0 & Tkinter==8.6
Whenever I enter some text and press button terminal shows lots of errors. Can you fix it?

**Comment by anshul7898** [comment]
yes sure can you please paste the error which is being shown up? you can paste the Screenshot of the error or error directly in the comments.
